variables.tf

```diff
@@ -0,0 +1,71 @@
+variable "motd_name" {
+  type        = string
+  default     = "jumpserver"
+  description = "Name of the place where the user joined. Defaults to 'jumpserver', so it shows: 'Welcome to jumpserver'"
+}
+
+variable "name" {
+  type        = string
+  default     = "jumpserver"
+  description = "Name of the resource. Defaults to 'jumpserver'"
+}
+
+variable "name_prefix" {
+  type        = string
+  default     = ""
+  description = "Prefix of the resource. If not specified it won't add a prefix."
+}
+
+variable "namespace" {
+  type        = string
+  default     = "default"
+  description = "Namespace where the resource will be deployed. If not specified it will be deployed in 'default' namespace."
+}
+
+variable "ssh_keys" {
+  type        = string
+  description = "List of SSH keys to be added to the authorized keys list."
```

Comment:
Maybe we need to clarify that it's a "list" in EOT format and that the variable type is not an oversight.